### DISCUSSION TITLE
How secure is ClickHouse® Cloud?

### DISCUSSION THREAD

**Discussion by emoido** [ChistaDATA/Cloud#11] [❓FAQ] [answered]
How secure is ClickHouse® Cloud?

**Comment by TheraniAA** [comment, marked as answer]
ChistaDATA Cloud places great emphasis on ensuring the security of ClickHouse® databases hosted on its platform. To achieve this goal, ChistaDATA has put in place a range of security measures, including data encryption for both data at rest and data in transit, access controls, network isolation, and continuous real-time monitoring. Furthermore, ChistaDATA offers multiple sources for authentication, such as GitHub authentication and two-factor authentication for the portal, in order to provide an additional layer of security to its users.
